packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -80,6 +83,11 @@ class CupertinoButton extends StatefulWidget {
   /// * <https://developer.apple.com/ios/human-interface-guidelines/visual-design/layout/>
   final double minSize;
 
+  /// The opacity that the button will fade to when it is active (pressed)
+  ///
+  /// This defaults to 0.1
```

Comment:
missing periods at the end of each line.

Also, you should talk about what the opacity will be when it's not being pressed.

Also, I recommend using "pressedOpacity" rather than "activeOpacity". The term "active" is heavily overloaded in UI design and could be confusing.